internal/e2e/flexible_test.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2015 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package e2e
+
+import (
+	"io/ioutil"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/aeintegrate"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestHelloWorld(t *testing.T) {
+	tc := testutil.EndToEndTest(t)
+
+	helloworld := &aeintegrate.App{
+		Name:      "hw",
+		Dir:       tc.Path("docs", "managed_vms", "helloworld"),
+		ProjectID: tc.ProjectID,
+	}
+
+	bodyShouldContain(t, helloworld, "/", "Hello world!")
+}
+
+func TestDatastore(t *testing.T) {
+	tc := testutil.EndToEndTest(t)
+
+	t.Skip("does not work with v1beta3 yet - internal bug 28008919")
+
+	datastore := &aeintegrate.App{
+		Name:      "ds",
+		Dir:       tc.Path("docs", "managed_vms", "datastore"),
+		ProjectID: tc.ProjectID,
+		Env: map[string]string{
+			"GCLOUD_DATASET_ID": tc.ProjectID,
+		},
+	}
+
+	bodyShouldContain(t, datastore, "/", "Succesfully stored")
+}
+
+func TestMemcache(t *testing.T) {
+	tc := testutil.EndToEndTest(t)
+
+	memcache := &aeintegrate.App{
+		Name:      "mem",
+		Dir:       tc.Path("docs", "managed_vms", "memcache"),
+		ProjectID: tc.ProjectID,
+	}
+
+	bodyShouldContain(t, memcache, "/", "Count")
+}
+
+func bodyShouldContain(t *testing.T, p *aeintegrate.App, path, shouldContain string) {
```

Comment:
I want it to be obvious that it's testing content, since we'll have some other kinds of tests later (e.g. ones that POST data)